lib/reek/examiner.rb

```diff
@@ -12,6 +12,19 @@ module Reek
   #
   # :reek:TooManyInstanceVariables: { max_instance_variables: 7 }
   class Examiner
+    INCOMPREHENSIBLE_SOURCE_TEMPLATE = <<-EOS.freeze
+      !!!
+      Source %s can not be processed by Reek.
+      This is most likely a Reek bug.
+      It would be great if you could report this back to the Reek team
+      by opening up a corresponding issue at https://github.com/troessner/reek/issues
+      Make sure to include the source in question, the Reek version
```

Comment:
Just a thought. Would it be feasible to have all this information ready to be copied and pasted from the error message, rather than having the user go and gather it?